indico/htdocs/sass/base/_layout.scss

```diff
@@ -137,22 +136,64 @@ body > * {
     }
 }
 
-.flex {
-    display: flex;
+/* Flexbox layout */
+.flexrow {
```

Comment:
Moving conversation from #2726 :
> ThiefMaster a day ago
> I still think this would be clearer:
> [18:37:52] <@ThiefMaster> flexrow -> .flex.row?
> [18:38:05] <@ThiefMaster> ie .flex to set display:flex and .col/.row to set the flow

> OmeGak 23 hours ago 
I don't think we should provide every class to be orthogonal with the full expressiveness of flex-boxes. flewrow` is just a class for an opinionated layout (flex row non-wrapping).